### PR TITLE
docs: rephrasing the hubble intro doc

### LINKED ISSUE #14587: Observability documentation duplicates text
Glancing through this page:
https://docs.cilium.io/en/v1.9/concepts/observability/intro/

The second and third paragraphs seem substantially similar, it seems like maybe a merge was mishandled to include multiple copies of the text. This affects the master version of the docs as well. It would be good to review the text and streamline it to avoid duplication.

### PR DESCRIPTION
**Pull request by vipul-21**
Please ensure your pull request adheres to the following guidelines:

- [ ] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)
- [ ] Thanks for contributing!

Rephrasing the doc to make sure to avoid duplication

Fixes: #14587

### CODE REVIEW

**Comment by vipul-21** [comment]
@joestringer can you take a look at this when you have time.

**Comment by joestringer** [comment]
@vipul-21 I don't review all PRs, GitHub has assigned CODEOWNERS for the relevant areas to review. As @cilium/tophat for the week I'll trigger CI for you :+1:

File: Documentation/gettingstarted/hubble_intro.rst [hunk 1]
```diff
@@ -17,18 +17,14 @@ node level, cluster level or even across clusters in a :ref:`Cluster Mesh`
...
 section :ref:`intro`.
 
-By default, the Hubble API is scoped to each individual node on which the
-Cilium agent runs. In other words, networking visibility is only provided for
-traffic observed by the local Cilium agent. In this scenario, the only way to
-interact with the Hubble API is by using the Hubble CLI (``hubble``) to query
-the Hubble API provided via a local Unix Domain Socket.  The Hubble CLI binary
-is installed by default on Cilium agent pods.
+By default, Hubble API operates within the scope of the individual node on which the 
+cilium agent runs. This confines the network insights to the traffic observed by the local
```
Comment (ghost) [inline Documentation/gettingstarted/hubble_intro.rst]:
```suggestion
Cilium agent runs. This confines the network insights to the traffic observed by the local
```

File: Documentation/gettingstarted/hubble_intro.rst [hunk 2]
```diff
@@ -17,18 +17,14 @@ node level, cluster level or even across clusters in a :ref:`Cluster Mesh`
...
+Cilium agent. Hubble CLI(``hubble``) can be used to query the Hubble API provided via a local
```
Comment (ghost) [inline Documentation/gettingstarted/hubble_intro.rst]:
```suggestion
Cilium agent. Hubble CLI (``hubble``) can be used to query the Hubble API provided via a local
```

File: Documentation/gettingstarted/hubble_intro.rst [hunk 3]
```diff
@@ -17,18 +17,14 @@ node level, cluster level or even across clusters in a :ref:`Cluster Mesh`
...
-scenario, the Hubble Relay service provides a Hubble API which scopes the
-entire cluster or even multiple clusters in a ClusterMesh scenario. Hubble data
-can be accessed by pointing a Hubble CLI (``hubble``) to the Hubble Relay
-service or via Hubble UI. Hubble UI is a web interface which enables automatic
-discovery of the services dependency graph at the L3/L4 and even L7 layer,
-allowing user-friendly visualization and filtering of data flows as a service
-map.
+Upon deploying Hubble Relay, network visibility is provided for the entire cluster or even
+multiple clusters in a ClusterMesh scenario. In this mode, Hubble data can be accessed by
+directing the Hubble CLI(``hubble``) to the Hubble Relay service or via Hubble UI.
```
Comment (ghost) [inline Documentation/gettingstarted/hubble_intro.rst]:
```suggestion
directing Hubble CLI (``hubble``) to the Hubble Relay service or via Hubble UI. 
```

File: Documentation/gettingstarted/hubble_intro.rst [hunk 4]
```diff
@@ -17,18 +17,14 @@ node level, cluster level or even across clusters in a :ref:`Cluster Mesh`
...
-By default, the Hubble API is scoped to each individual node on which the
-Cilium agent runs. In other words, networking visibility is only provided for
-traffic observed by the local Cilium agent. In this scenario, the only way to
-interact with the Hubble API is by using the Hubble CLI (``hubble``) to query
-the Hubble API provided via a local Unix Domain Socket.  The Hubble CLI binary
-is installed by default on Cilium agent pods.
+By default, Hubble API operates within the scope of the individual node on which the 
+cilium agent runs. This confines the network insights to the traffic observed by the local
+Cilium agent. Hubble CLI(``hubble``) can be used to query the Hubble API provided via a local
+Unix Domain Socket. (The Hubble CLI binary is installed by default on Cilium agent pods.)
```
Comment (ghost) [inline Documentation/gettingstarted/hubble_intro.rst]:
```suggestion
Unix Domain Socket. The Hubble CLI binary is installed by default on Cilium agent pods.
```
Is there a good reason for using parentheses?

**Review by ghost — CHANGES_REQUESTED** [review]
Thanks @vipul-21, I left a few suggestions. Could you also put reasoning for the change in the commit message? Based on the linked issue, I understand the change is done to reduce duplication between the two paragraphs.

**Review by chancez — APPROVED** [review]

**Review by chancez — CHANGES_REQUESTED** [review]
Can you squash your commits together?

**Review by chancez — APPROVED** [review]
Thanks!

**Review by ghost — APPROVED** [review]

**Pull request merged**